__tests__/lib/api/http-client.test.ts

```
@@ -209,3 +209,64 @@ describe("HttpClient with FetchAdapter", () => {
+  afterEach(() => {
+    // Restore original environment
+    process.env = { ...originalEnv };
+  });
+
+  it("should create authConfig when both username and password are provided", () => {
+    // Set environment variables
+    process.env.NEXT_PUBLIC_API_USERNAME = "testuser";
+    process.env.NEXT_PUBLIC_API_PASSWORD = "testpass";
+
+    // Re-import to get fresh instance with new env vars
+    const { apiClient } = require("../../../src/lib/api/http-client");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)